includes/CPT/class-issue.php

```diff
@@ -0,0 +1,252 @@
+<?php
```

Comment:
This is very duplicate of class-profile.php. Construct a class-post-type.php base class and inherit from that.

Reply:
This has been done now.